ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -276,3 +276,33 @@ def gather(
         "batch_dims": batch_dims
     }
     return _get_node_factory_opset8().create("Gather", inputs, attributes)
+
+
+@nameable_op
+def random_uniform(
+        output_shape: NodeInput,
+        min_val: NodeInput,
+        max_val: NodeInput,
+        output_type: str,
+        global_seed: int = 0,
```

Comment:
Is `global_seed` name a planned change? Is there a check in C++ implementation that ensure integers are positive? Moreover there is no info about default value.

Docs spec:
https://github.com/openvinotoolkit/openvino/blame/master/docs/ops/generation/RandomUniform_8.md#L178-L183

Reply:
Yes, it is a planned change which was performed during review of [shell implementation](https://github.com/openvinotoolkit/openvino/blob/2793963e6f759fc6ca89130d1403eacd2778d6b2/ngraph/core/include/openvino/op/random_uniform.hpp#L28). I will correct specification to align with new attribute names and will add default values. 

In c++ implementation uint64_t type is used for seed values, so values are not negative.

Reply:
Great! Please edit specification as soon as possible 🚀 

Anyway for Python it means that we can pass any integer. Can you add a note to both of seeds that it is required to pass a positive number? Also a check that throws error/warning for `< 0` would be nice.

```
@param global_seed: Specifies global seed value. Required to be a positive integer or 0.
```

Reply:
Added checks and a note.
Here is the PR with specification update: https://github.com/openvinotoolkit/openvino/pull/7491